public/react-services/generic-request.js

```diff
@@ -102,6 +105,7 @@ export class GenericRequest {
           if (!window.location.hash.includes('#/settings')) {
             window.location.href = getHttp().basePath.prepend('/app/wazuh#/health-check');
           }
+          throw new Error('Current API is down', err);
```

Comment:
**issue**: the Error class only receive an argument in its constructor

**suggestion**: Remove the second argument or integrate the error variable/message in the first argument.